job/src/run_jump_model.py

```diff
@@ -89,9 +89,9 @@ def run_jump_model(run_uuid):
         t_start = time.time()
         julia_host = os.environ.get('JULIA_HOST', "julia")
         response = requests.post("http://" + julia_host + ":8081/reopt/", json=data)
-        if response.status_code == 500:
-            raise REoptFailedToStartError(task=name, message=results["error"], run_uuid=run_uuid, user_uuid=user_uuid)
         response_json = response.json()
+        if response.status_code == 500:
+            raise REoptFailedToStartError(task=name, message=response_json["error"], run_uuid=run_uuid, user_uuid=user_uuid)
```

Comment:
@rathod-b any reason for this switch?  It seems like the error check on `response` should happen before creating `response_json` as the original order had it

Reply:
Ah, I see it now - the response_json["error"] is in the correct format.